### PR TITLE
End-To-End hc deployment

### PR DESCRIPTION
**Pull request by gobindadas**
@sac Please review and merge.

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -0,0 +1,19 @@
+hc-ansible-deployment is end-to-end HC deployment playbook which is using both gluster-ansible role and ovirt-ansible-hosted-engine-setup role.
```
Comment (sac) [inline README]: Gobinda we need to re-organize this a bit. Putting the playbooks in a directory called playbooks/hc/

File: README [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+1. There is one inventory file which is gluster_inventory.yml/gluster_inventory_vdo.yml has gluster related configurations
```
Comment (sabose) [inline README]: why do we need 2 inventory files? Can the vdo variables be captured in gluster_inventory as well (commented out if not needed)?

File: README [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+2. The json file he_gluster_vars.json has Hosted-Engine related configurations for gluster.
+
+For new HC deployment both file need to be modified with accurate values like host,device,pvs,vgs,lvs,fqdn,passwords etc
+
+How to run:
```
Comment (sabose) [inline README]: want to create the step before of yum installing the role rpms?

File: README (hunk 1 above)
Comment (gobindadas) [reply README]: Done.

File: README (hunk 2 above)
Comment (gobindadas) [reply README]: Included vdo configs in gluster_inventory and removed gluster_inventory_vdo.yml

File: README [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+ > cd hc-ansible-deployment
+
+ With VDO:
+ > ansible-playbook -i gluster_inventory_vdo.yml hc_deployment.yml --extra-vars='@he_gluster_vars.json'
+
+ Without VDO:
+ > ansible-playbook -i gluster_inventory.yml hc_deployment.yml --extra-vars='@he_gluster_vars.json'
+
```
Comment (sac) [inline README]: @gobindadas you might want to change the documentation. Now that you have removed the vdo inventory.

File: README [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+For gluster deployment clean up:
+ > ansible-playbook -i gluster_inventory_vdo.yml/gluster_inventory.yml tasks/gluster_cleanup.yml
+
```
Comment (sac) [inline README]: Same here.

File: README (hunk 4 above)
Comment (gobindadas) [reply README]: @sac This is already taken care.

File: README (hunk 5 above)
Comment (gobindadas) [reply README]: @sac This is already taken care.

File: README (hunk 4 above)
Comment (gobindadas) [reply README]: @sac It's no longer required, so removed it.

**Pull request merged**